Formula/mk-configure.rb

```diff
@@ -0,0 +1,22 @@
+class MkConfigure < Formula
+  desc "Lightweight replacement for GNU autotools"
+  homepage "https://github.com/cheusov/mk-configure"
+  url "https://downloads.sourceforge.net/project/mk-configure/mk-configure/mk-configure-0.29.3/mk-configure-0.29.3.tar.gz"
+  sha256 "1f483f717a09d06c82a6f3f284dd34234f54ead032e6987d33f3cf8109490c05"
+
+  depends_on "bmake"
+  depends_on "makedepend"
+
+  def install
+    ENV["PREFIX"] = prefix
+    ENV["MANDIR"] = man
+
+    system "bmake", "all"
+    system "bmake", "install"
+    doc.install("presentation/presentation.pdf")
```

Comment:
Let's drop the `()` here. Just `doc.install "presentation/presentation.pdf"`.

Wouldn't have pointed it out at this point normally but since the previous CI build is mysteriously red this will need to go through testing again to see what that's about, and pushing another commit will achieve that.